REFERENCE.md

```diff
@@ -0,0 +1,195 @@
+# Learnosity Node.js-SDK: Reference guide
+
+## Examples
+You can find a complete site with examples of Learnosity integration in our [demo site](http://demos.learnosity.com/).
+
+You can download the entire site or browse the code directly on [GitHub](https://github.com/Learnosity/learnosity-demos/).
+
+## Usage
+
+### Learnosity.init()
```

Comment:
Also that `init()` creates a new request object each time, so can be called as many times as you want. It is not very well named, since it is not really initialising anything.

Reply:
For discussion.

Reply:
I'd like to add this information to our more instructional article idea.